examples/nodejs/observability/package.json

```diff
@@ -29,8 +29,8 @@
     "typescript": "4.4.3"
   },
   "dependencies": {
-    "@gomomento/sdk": "1.26.2",
-    "@grpc/grpc-js": "1.8.14",
+    "@gomomento/sdk": "1.28.0",
+    "@grpc/grpc-js": "1.8.17",
```

Comment:
this looks fine but i'm wondering if for the grpc-js one, if we used one of those fuzzy match things like `^1.whatever`, if that would make it less likely to hit this in the future. I genuinely don't know; probably not, if the version gets locked in package-lock.json anyway.

not a blocker either way, just food for thought.

Reply:
Yeah, since we're shipping `package-lock` I also don't think that would help here. If this becomes a recurrent issue, we should think about maybe relaxing some of these selectors and removing the lockfiles for the examples.